run.py

```diff
@@ -346,13 +349,23 @@ def run(args):
 
     # Mandatory arguments
     rhbuild = args["--rhbuild"]
-    glb_file = args.get("--global-conf")
-    if not glb_file:
-        glb_file = args.get("--cluster-conf")
-    inventory_file = args["--inventory"]
-    osp_cred_file = args["--osp-cred"]
     suite_files = args["--suite"]
+
+    glb_file = args.get("--global-conf")
+    if args.get("--cluster-conf"):
+        glb_file = args["--cluster-conf"]
+
+    # Deciders
+    reuse = args.get("--reuse", None)
```

Comment:
by default can be `--store`.

Reply:
It is something we can ponder about. `--store` and `--reuse` are optional hence setting to default `--store` can be change the behavior unless we want to.